XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -825,7 +741,7 @@ A JSON object representing a dictionary of accounts to CFToken objects. Includes
 Used to continue querying where we left off when paginating. Omitted if there are no more entries after this result.
 
  ### 1.7 Free CFTs
-When a holder creates a `CFTokenPage`, if the holder owns at most 2 items in the ledger including the new page, the account's owner reserve is treated as zero instead of the normal amount. This is following the status quo of how free trustlines work today.
+When a holder creates a `CFToken`, if the holder owns at most 2 items in the ledger including the new `CFToken`, the account's owner reserve is treated as zero instead of the normal amount. This is following the status quo of how free trustlines work today.
```

Comment:
I think we should strike this section. In theory, we can propose anything we'd like in this specification.

However, ledger reserves exist to protect the operators from having to store free data on behalf of ledger users. Making a "free tier" go against that idea, and shouldn't be modified on a per-feature basis. 

Instead, fee changes should be an XLS of their own, and independent of any particular feature. Not having this free tier will also mean we have one less thing to debate when or if this spec ever gets put up for an ammendment.

Last but not least, this free tier doesn’t seem to be necessary because a user must “opt in” to hold CFTs anyway, so that user can decide at that point if they want to pay the 2XRP reserve or not (this isn't a very large amount to bear anyway).

I'm also happy to move this one change to a different PR if my suggestion is controversial...?

Reply:
I'll propose this in a different PR.